frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx

```diff
@@ -75,10 +81,7 @@ export default class SettingsUpdatesForm extends Component {
         </div>
       );
     } else {
-      return (
-        <div>{t`Sorry, we were unable to check for updates at this time. Last successful check was
-         ${MetabaseSettings.versionInfoLastChecked()}.`}</div>
-      );
+      return <div>{t`No successful checks yet.`}</div>;
```

Comment:
JFYI, we shouldn't introduce a new string (for i18n) in the last minute before v40 release.

Reply:
@ariya Yes, I was going to confirm this for this specific situation. Thanks!

Reply:
Increasingly thinking that getting to the bottom of this i18n flow and being able to quickly turn around translations _might_ make us way faster and less prone to dropping balls in the 3-month-Metabase-zen-time-horizon.

Aware of the large queue of priorities we have, not to worry.

Reply:
Separate ticket for this, to be done after v40.

https://github.com/metabase/metabase/issues/16923

Queued in board https://github.com/orgs/metabase/projects/16

Reply:
Updated again, Maz is 👍 for this situation: https://github.com/metabase/metabase/pull/16920#discussion_r665731675